nuget/Auth0.OidcClient.UWP.nuspec

```diff
@@ -46,15 +46,15 @@
     <copyright>Copyright 2017-2019 Auth0, Inc.</copyright>
```

Comment:
What is protected by copyright here? 🤔

Reply:
That's the copyright notice that appears in the package you see on the package managers.